saleor/payment/gateways/np_atobarai/__init__.py

```diff
@@ -0,0 +1,156 @@
+import logging
+import os
+from typing import List
+
+from ....order.models import Fulfillment
+from ... import PaymentError, TransactionKind
+from ...interface import GatewayConfig, GatewayResponse, PaymentData
+from ...models import Payment
+from . import api
+from .api_types import ApiConfig, PaymentStatus, get_api_config
+from .utils import (
+    get_fulfillment_for_order,
+    get_payment_name,
+    get_shipping_company_code,
+    notify_dashboard,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def inject_api_config(fun):
+    def inner(payment_information: PaymentData, config: GatewayConfig):
+        return fun(payment_information, get_api_config(config.connection_params))
+
+    return inner
+
+
+def parse_errors(errors: List[str]) -> str:
+    # FIXME: better solution?
+    #  Transaction.error in database has max_length of 256
+    return os.linesep.join(errors)[:256]
+
+
+@inject_api_config
+def process_payment(
+    payment_information: PaymentData, config: ApiConfig
+) -> GatewayResponse:
+    payment_id = payment_information.payment_id
+    payment = Payment.objects.filter(pk=payment_id).first()
+
+    if not payment:
+        raise PaymentError(f"Payment with id {payment_id} does not exist.")
+
+    result = api.register_transaction(payment.order, config, payment_information)
+
+    return GatewayResponse(
+        is_success=result.status == PaymentStatus.SUCCESS,
+        action_required=False,
+        kind=TransactionKind.CAPTURE,
+        amount=payment_information.amount,
+        currency=payment_information.currency,
+        transaction_id=result.psp_reference,
+        error=parse_errors(result.errors),
+        raw_response=result.raw_response,
+        psp_reference=result.psp_reference,
+    )
+
+
+@inject_api_config
+def capture(payment_information: PaymentData, config: ApiConfig) -> GatewayResponse:
+    raise NotImplementedError
+
+
+@inject_api_config
+def void(payment_information: PaymentData, config: ApiConfig) -> GatewayResponse:
+    raise NotImplementedError
+
+
+@inject_api_config
+def tracking_number_updated(fulfillment: Fulfillment, config: ApiConfig) -> None:
+    from .plugin import NPAtobaraiGatewayPlugin
```

Comment:
You're importing this to only get a plugin_id. Can't we move plugin ID do `const.py` file and import it normaly in this file and in the `plugin.py`? 
In `NPAtobaraiGatewayPlugin` you could just use simple assigment `PLUGIN_ID=const.PLUGIN_ID`